pkg/orchestrator/azureDevOps.go

```diff
@@ -29,6 +29,57 @@ func (a *AzureDevOpsConfigProvider) InitOrchestratorProvider(settings *Orchestra
 	log.Entry().Debug("Successfully initialized Azure config provider")
 }
 
+func (a *AzureDevOpsConfigProvider) getAPIInformation() {
```

Comment:
Getter usually return something. I would rename it to fetch or return the api information